<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportWord</class>
 <widget class="QMainWindow" name="ImportWord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-20</y>
      <width>511</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HGH2X_CNKI</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Please import some words first!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>500</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Handwriting</family>
      <pointsize>16</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>741</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionRestart"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
